cmake/modules/SwiftSupport.cmake

```diff
@@ -13,7 +13,7 @@ function(add_swift_library library)
   list(APPEND flags -emit-library)
 
   if(ASL_TARGET)
-    list(APPEND FLAGS -target;${ASL_TARGET})
+    list(APPEND flags -target;${ASL_TARGET})
```

Comment:
apparently CMake variable names are case sensitive. This is a typo introduced in an earlier PR